py-polars/polars/utils/show_versions.py

```diff
@@ -58,18 +59,20 @@ def show_versions() -> None:
 def _get_dependency_info() -> dict[str, str]:
     # see the list of dependencies in pyproject.toml
     opt_deps = [
-        "adbc_driver_sqlite",
+        "adbc_driver_manager",
```

Comment:
Why was this changed from `adbc_driver_sqlite`? Now it doesn't match `pyproject.toml`.

Reply:
Ahh, this appears to be my (unfinished) PR for ADBC fixes accidentally leaking into this commit - will fix it up 😅